functional-tests/tests/plain_highlight.js

```diff
@@ -25,6 +29,28 @@ After(function() {
 Scenario('Create/Delete a new plain highlight annotation @desktop @doc', function(I) {
     I.waitForVisible(SELECTOR_ANNOTATIONS_LOADED);
 
+    /** Highlighting should still work after drawing on the file */
```

Comment:
Would it make sense to break this out into a separate scenario, "highlight after drawing on the file", and leave the simple scenario of creating a plain highlight?

Reply:
Breaking it out into a separate Scenario causes the page to refresh. Apparently you can't open the page in the BeforeSuite helper which would have allowed me to break each segment into it's own scenario :-/